heartbeat/docs/getting-started.asciidoc

```diff
@@ -1,184 +1,73 @@
-[[heartbeat-getting-started]]
-== Get started with {beatname_uc}
+[id="{beatname_lc}-installation-configuration"]
+== {beatname_uc} quick start: installation and configuration
 
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Quick start: installation and configuration</titleabbrev>
```

Comment:
Why "Installation and configuration" instead of "Install and configure"?

Reply:
I checked with Anne B, and those terms rank very high in google searches.